doc/classes/Node3D.xml

```diff
@@ -232,143 +239,159 @@
+			- The [member Vector3.x] is the angle around the local X axis (pitch);
+			- The [member Vector3.y] is the angle around the local Y axis (yaw);
+			- The [member Vector3.z] is the angle around the local Z axis (roll).
+			The order of each consecutive rotation can be changed with [member rotation_order] (see [enum EulerOrder] constants). By default, the YXZ convention is used ([constant EULER_ORDER_YXZ]).
+			[b]Note:[/b] This property is edited in degrees in the inspector. If you want to use degrees in a script, use [member rotation_degrees].
 		</member>
 		<member name="rotation_degrees" type="Vector3" setter="set_rotation_degrees" getter="get_rotation_degrees">
-			Helper property to access [member rotation] in degrees instead of radians.
+			The [member rotation] of this node, in degrees instead of radians.
+			[b]Note:[/b] This is [b]not[/b] the property available in the Inspector dock.
 		</member>
 		<member name="rotation_edit_mode" type="int" setter="set_rotation_edit_mode" getter="get_rotation_edit_mode" enum="Node3D.RotationEditMode" default="0">
-			Specify how rotation (and scale) will be presented in the editor.
+			How this node's rotation and scale are displayed in the Inspector dock.
 		</member>
 		<member name="rotation_order" type="int" setter="set_rotation_order" getter="get_rotation_order" enum="EulerOrder" default="2">
-			Specify the axis rotation order of the [member rotation] property. The final orientation is constructed by rotating the Euler angles in the order specified by this property.
+			The axis rotation order of the [member rotation] property. The final orientation is calculated by rotating around the X, Y, and Z axis in this order.
 		</member>
 		<member name="scale" type="Vector3" setter="set_scale" getter="get_scale" default="Vector3(1, 1, 1)">
-			Scale part of the local transformation.
-			[b]Note:[/b] Mixed negative scales in 3D are not decomposable from the transformation matrix. Due to the way scale is represented with transformation matrices in Godot, the scale values will either be all positive or all negative.
-			[b]Note:[/b] Not all nodes are visually scaled by the [member scale] property. For example, [Light3D]s are not visually affected by [member scale].
+			Scale of this node in parent space (relative to the parent node). This value is obtained from [member basis]'s scale.
+			[b]Note:[/b] The behavior of some 3D node types is not affected by this property. These include [Light3D], [Camera3D], [AudioStreamPlayer3D], etc.
+			[b]Warning:[/b] The scale's components must either be all positive or all negative, and [b]not[/b] exactly [code]0.0[/code]. Otherwise, it won't be possible to obtain the scale from the [member basis]. This may cause the intended scale to be lost when reloaded from disk, and potentially other unstable behavior.
```

Comment:
Alternatives to the same note. This is the same as the original, but switched around:
```suggestion
			[b]Warning:[/b] Due to the way scale is represented with transformation matrices in Godot, the scale's components must either be all positive, all negative, or all [code]0.0[/code]. Mixed negative scales in 3D cannot be decomposed from the [member transform].
```
This... I don't know
```suggestion
			[b]Warning:[/b] The scale's components must either be all positive or all negative, and [b]not[/b] exactly [code]0.0[/code]. Otherwise, this may cause unstable behavior, as it won't be possible to obtain the intended scale from the [member basis].
```

Reply:
Doesn't the first alternative invert the meaning...? I thought that scale *shouldn't* be all `0.0`, as per this note. You can check this trivially in the editor, you get `Condition "det == 0" is true.` errors.